framework/integration/test/basic/basic_test.go

```diff
@@ -28,15 +33,18 @@ func Test_Finalizer_Integration_Basic(t *testing.T) {
 		Name:      operatorName,
 		Namespace: testNamespace,
 	}
-
-	client.MustSetup(testNamespace)
-	defer client.MustTeardown(testNamespace)
-
-	operatorkitFramework, err := client.NewFramework(c)
+	configmapClient, err := configmap.New(c)
```

Comment:
These 5 lines should be part of a function which is declared for example in the testdata of a table test. That way we could just deal with `client.Interface` inside the testcase.